src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,120 @@
+import { FC, useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { preparePeople, getCentury } from '../../helper';
+import { SearchParams } from '../../types/SearchParams';
+import { Sort } from '../../types/Sort';
+import { Person } from '../../types/Person';
+import { PeopleFilters } from '../PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [searchParams] = useSearchParams();
+
+  let currentPeople = [...people];
+
+  const sexParams = searchParams.get(SearchParams.Sex);
+  const queryParams = searchParams.get(SearchParams.Query);
+  const centuriesParams = searchParams.getAll(SearchParams.Centurie);
+  const sortParams = searchParams.get(SearchParams.Sort);
+  const orderParams = searchParams.get(SearchParams.Order) === 'desc';
+
+  if (sexParams) {
+    currentPeople = currentPeople.filter(({ sex }) => sex === sexParams);
+  }
+
+  if (centuriesParams.length > 0) {
+    currentPeople = currentPeople.filter(
+      person => centuriesParams.includes(
+        getCentury(person).toString(),
+      ),
+    );
+  }
+
+  if (queryParams) {
```

Comment:
Create helper functions for filtering and move them outside the component to utils/helpers file or folder